### PR TITLE
[BEAM-9342[ Update bytebuddy to version 1.10.8

### PR DESCRIPTION
**Pull request by iemejia**
R: @kennknowles @lukecwik

### CODE REVIEW

File: sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoTest.java [hunk 1]
```diff
@@ -514,9 +514,7 @@ public void proccessElement(ProcessContext c) {}
       SingleOutput<KV<String, String>, String> parDo = ParDo.of(fn);
 
       // Use the parDo in a pipeline to cause state coders to be inferred.
-      pipeline
-          .apply(Create.of(KV.of("input", "value")))
-          .apply(parDo);
+      pipeline.apply(Create.of(KV.of("input", "value"))).apply(parDo);
```
Comment (iemejia) [inline sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoTest.java]: changes in this class are unrelated but auto applied by `spotlessApply`

**Comment by lukecwik** [comment]
Run Python PreCommit

**Comment by lukecwik** [comment]
Run Portable_Python PreCommit

**Comment by lukecwik** [comment]
Run Go PreCommit

**Comment by iemejia** [comment]
Rebased to catch with the fix for the formatting error #11010

**Comment by iemejia** [comment]
Run Java PreCommit

**Comment by iemejia** [comment]
Merging eagerly, the python error is unrelated to this PR and related to some missing import on Gcp related tests.

**Pull request merged**